core/frontend/src/ViewCreator3d.ts

```diff
@@ -138,7 +138,7 @@ export class ViewCreator3d {
     };
 
     const cameraData = new Camera();
-    const cameraOn = options?.cameraOn ? options.cameraOn : false;
+    const cameraOn = options?.cameraOn !== false;
```

Comment:
In ViewCreator3dOptions, document this new default. People find it surprising when "undefined" defaults to "true".

Reply:
@pmconne  - updated comments